docs/components/avatar/index.md

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
I'd suggest moving these files to the component folder and using the scripts/build-docs.sh to copying it over.  At least then we're managing the content at the source.

Reply:
Let's also add this to the generator after moving it so that new components come with this template started for them.

Reply:
I think we need to take a nice, large look at all of this. The documentation pages for the components is all manually created at the moment and I imagine we could do something with JSDoc to automate the creation of the docs pages. For the purposes of this PR, I'd like to just get this merged so we can shut down the older site.

Reply:
Okay, I added an issue for it so we can come back to it later.